cpp/example_code/eventbridge/put_rule.cpp

```diff
@@ -1,28 +1,16 @@
- 
-//snippet-sourcedescription:[put_rule.cpp demonstrates how to create an Amazon CloudWatch Events routing rule.]
-//snippet-keyword:[C++]
-//snippet-sourcesyntax:[cpp]
-//snippet-keyword:[Code Sample]
-//snippet-keyword:[Amazon CloudWatch Events]
-//snippet-service:[events]
-//snippet-sourcetype:[full-example]
-//snippet-sourcedate:[]
-//snippet-sourceauthor:[AWS]
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
 
-
-/*
-   Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-
-   This file is licensed under the Apache License, Version 2.0 (the "License").
-   You may not use this file except in compliance with the License. A copy of
-   the License is located at
-
-    http://aws.amazon.com/apache2.0/
-
-   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-   CONDITIONS OF ANY KIND, either express or implied. See the License for the
-   specific language governing permissions and limitations under the License.
-*/
+/* ////////////////////////////////////////////////////////////////////////////
+ * Purpose: put_rule.cpp demonstrates how to create an Amazon CloudWatch Events routing rule.
+ *
+ * Inputs:
+ * - rule_name: The name of the rule.
+ * - role_arn: The Amazon Resource Name (ARN) of the IAM role associated with the rule.
+ *
+ * Outputs:
+ * The routing rule is created.
+   * ///////////////////////////////////////////////////////////////////////// */
 //snippet-start:[cw.cpp.put_rule.inc]
 #include <aws/core/Aws.h>
 #include <aws/events/CloudWatchEventsClient.h>
```

Comment:
This example must be updated to use EventBridgeClient instead of CloudWatchEventsClient.

Reply:
fixed